operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md

```diff
@@ -34,13 +41,13 @@ Only follow the steps in the section for the node type that is being rebuilt.
 
 #### Option 1
 
-(`ncn-m001#`) Run `ncn-upgrade-worker-storage-nodes.sh` for `ncn-w001`.
+1. (`ncn-m001#`) Run `ncn-upgrade-worker-storage-nodes.sh` for `ncn-w001`.
```

Comment:
We try to avoid numbered lists that have just one item.

```suggestion
(`ncn-m001#`) Run `ncn-upgrade-worker-storage-nodes.sh` for `ncn-w001`.
```